### PR TITLE
Scheduled Updates: Unlock the health path check feature

### PR DESCRIPTION
**Pull request by ouikhuan**
Related to https://github.com/Automattic/dotcom-forge/issues/6531

## Proposed Changes

* Unlock the health check path feature to all the users.

## Testing Instructions

* Code review should be enough, but if you want to double check, apply this PR and make sure the health check path feature shows up correctly when create/edit a schedule.

## Pre-merge Checklist

- [ ] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-aUh-p2)?

### CODE REVIEW

File: client/blocks/plugins-scheduled-updates/schedule-form.tsx [hunk 1]
```diff
@@ -162,17 +161,14 @@ export const ScheduleForm = ( props: Props ) => {
...
-				<>
-					<Text>{ translate( 'Step 3' ) }</Text>
-					<ScheduleFormPaths
-						paths={ healthCheckPaths }
-						borderWrapper={ false }
-						onChange={ setHealthCheckPaths }
-					/>
-				</>
-			) }
+			<>
```
Comment (bogiii) [inline client/blocks/plugins-scheduled-updates/schedule-form.tsx]: In this case we don't need the Fragment wrapper anymore.

**Review by bogiii — APPROVED** [review]

File: client/blocks/plugins-scheduled-updates/schedule-form.tsx (hunk 1 above)
Comment (ouikhuan) [reply client/blocks/plugins-scheduled-updates/schedule-form.tsx]: True, removed. Thank you!

File: client/blocks/plugins-scheduled-updates/schedule-form.tsx [hunk 2]
```diff
@@ -163,16 +162,12 @@ export const ScheduleForm = ( props: Props ) => {
 				} }
 			/>
 
-			{ isEnabled( 'plugins/multisite-scheduled-updates' ) && (
```
Comment (obenland) [inline client/blocks/plugins-scheduled-updates/schedule-form.tsx:166]: do we need to remove the flag from config files?

Comment (bogiii) [reply client/blocks/plugins-scheduled-updates/schedule-form.tsx:166]: Not yet; we still use it in the router configuration, hiding the multisite schedule management.

**Pull request merged**